### PR TITLE
data: Abhish7k

### PR DESCRIPTION
**Pull request by Abhish7k**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7961"></a>

### CODE REVIEW

**Comment by Vishrut19** [comment]
Thanks for contributing. LGTM 👍

File: data/Abhish7k.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Abhishek",
+  "bio": "Full Stack Developer",
+  "links": [
+    {
```
Comment (kumarsonsoff3) [inline data/Abhish7k.json:5]: Please add the `name` key of the link. For `links` it's necessary to use `name` key. 

```suggestion
  "links": [
   {
  "name": "Follow me on GitHub",
```

 Or modify it to use as socials on your profile

```suggestion
  "socials": [
    {
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Comment by Abhish7k** [comment]
Thank you 😁

**Pull request merged**